public/js/custom-webtrends.js

```diff
@@ -1,14 +1,18 @@
-(function(global){
+(function (global) {
   "use strict"
   var $ = global.jQuery;
   var document = global.document;
 
   $(document).ready(function () {
     $('.details__summary').on('touchstart click', function () {
-      global.Webtrends.multiTrack({argsa: ['DCSext.GPAppointment', 'Not_Registered', 'WT.dl', '121']});
+      if (global.Webtrends) {
```

Comment:
Can a comment be added here to explain why the check is being done at this point rather than during the document ready function? I think I might look back on this and wonder why it was done within both click handlers and be tempted to refactor it and undo the fix.